cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -29,7 +29,6 @@ import (
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/phases/certs"
-	authzmodes "k8s.io/kubernetes/pkg/kubeapiserver/authorizer/modes"
```

Comment:
does that mean that https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/.import-restrictions is currently not working?

`/hack/verify-import-boss.sh`

i do not see `pkg/kubeapiserver/authorizer/modes` as an expection there yet the import existing in kubeadm.

Reply:
It looks like import-boss doesn't care about _test.go files. Here is the quote from it's log demonstrating this:
```
I0809 12:27:45.341522   19053 parse.go:287] addDir k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane
I0809 12:27:45.630471   19053 parse.go:139] saving buildPackage k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane
I0809 12:27:45.630547   19053 parse.go:182] addFile k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane /home/ed/go/src/k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane/manifests.go
I0809 12:27:45.632103   19053 parse.go:182] addFile k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane /home/ed/go/src/k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane/volumes.go
I0809 12:27:45.633160   19053 parse.go:352] importPackage k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane, canonical path is k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane
I0809 12:27:45.633176   19053 parse.go:392] typeCheckPackage k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane
I0809 12:27:45.633216   19053 parse.go:327] importPackage fmt
I0809 12:27:45.633234   19053 parse.go:333] importPackage fmt, canonical path is fmt
I0809 12:27:45.633246   19053 parse.go:392] typeCheckPackage fmt
I0809 12:27:45.633259   19053 parse.go:395] typeCheckPackage fmt already done
I0809 12:27:45.633279   19053 parse.go:327] importPackage net
I0809 12:27:45.633302   19053 parse.go:333] importPackage net, canonical path is net
I0809 12:27:45.633315   19053 parse.go:392] typeCheckPackage net
I0809 12:27:45.633339   19053 parse.go:395] typeCheckPackage net already done
I0809 12:27:45.633354   19053 parse.go:327] importPackage os
I0809 12:27:45.633371   19053 parse.go:333] importPackage os, canonical path is os
I0809 12:27:45.633381   19053 parse.go:392] typeCheckPackage os
I0809 12:27:45.633393   19053 parse.go:395] typeCheckPackage os already done
I0809 12:27:45.633408   19053 parse.go:327] importPackage path/filepath
I0809 12:27:45.633423   19053 parse.go:333] importPackage path/filepath, canonical path is path/filepath
I0809 12:27:45.633433   19053 parse.go:392] typeCheckPackage path/filepath
I0809 12:27:45.633443   19053 parse.go:395] typeCheckPackage path/filepath already done
I0809 12:27:45.633453   19053 parse.go:327] importPackage strconv
I0809 12:27:45.633461   19053 parse.go:333] importPackage strconv, canonical path is strconv
...
```

Reply:
@neolit123 ^^^

Reply:
yeah, already tracked here:
https://github.com/kubernetes/kubernetes/issues/81102